.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: required
 
-language: scala
```

Comment:
Ah. I got it from here: https://github.com/azavea/usace-program-analysis-geoprocessing/blob/develop/.travis.yml